src/v/pandaproxy/api/api-doc/schema_registry_definitions.def.json

```diff
@@ -79,15 +79,37 @@
       "type": "object",
       "properties": {
         "compatibilityLevel": {
-          "type": "string"
+          "type": "string",
+          "description": "Compatibility Level",
```

Comment:
```suggestion
          "description": "Compatibility level",
```
nit; we usually try to default to sentence casing in the docs if it's not a page title